parsons/scytl/scytl.py

```diff
@@ -0,0 +1,664 @@
+import urllib
+import zipfile
+import csv
+import requests
+import urllib.error
+import urllib.request
+import xml.etree.ElementTree as ET
+from datetime import datetime
+from dateutil.parser import parse as parsedate
+from pytz import timezone
+from io import BytesIO, StringIO
+
+clarity_url = 'https://results.enr.clarityelections.com/'
+
+get_version_url = \
+    clarity_url + '{administrator}/{election_id}/current_ver.txt'
+summary_csv_url_template = \
+    clarity_url + '{administrator}//{election_id}/{version_num}/reports/summary.zip'
+detail_xml_url_template = \
+    clarity_url + '{administrator}/{election_id}/{version_num}/reports/detailxml.zip'
+county_detail_xml_url = clarity_url + \
+    '{state}/{county_name}/{county_election_id}/{county_version_num}/reports/detailxml.zip'
+detail_settings_json_url_template = \
+    clarity_url + '{state}/{election_id}/{version_num}/json/en/electionsettings.json'
+
+tzinfos = {
+    "EST": "UTC-5",
+    "EDT": "UTC-4",
+    "CST": "UTC-6",
+    "CDT": "UTC-5",
+    "MST": "UTC-7",
+    "MDT": "UTC-6",
+    "PST": "UTC-8",
+    "PDT": "UTC-7",
+    "AKST": "UTC-9",
+    "AKDT": "UTC-8",
+    "HST": "UTC-10",
+    "HDT": "UTC-9",
+}
+
+
+class CountyDetails:
+    def __init__(
+        self,
+        state: str,
+        county_name: str,
+        county_election_id: str,
+        county_version_num: str,
+        county_update_date: datetime = None
+    ):
+        self.state = state
+        self.county_name = county_name
+        self.county_election_id = county_election_id
+        self.county_version_num = county_version_num
+        self.county_update_date = county_update_date
+
+
+class Scytl(object):
```

Comment:
Why the `object` is correct (and the convention of the codebase), it also is unnecessary.